boards/rak/rak11720/doc/index.rst

```diff
@@ -0,0 +1,122 @@
+.. _rak11720:
+
+RAK11720 LoRa Module
+####################
+
+The RAK11720 is a WisBlock Core module for RAK WisBlock.
+It is based on the powerful ultra-low power Apollo3 Blue SoC (AMA3B1KK-KBR-B0)
+from Ambiq together with a Semtech SX1262 LoRa® transceiver.
+
+The AMA3B1KK-KBR-B0 has an integrated Bluetooth Low Energy transceiver
+that enhances the communication capabilities. The RAK11720 stamp module
+comes in the same size and footprint as our RAK3172 module which gives
+you the opportunity to enhance your existing designs
+with BLE without designing a new PCB.
+
+.. image:: ./rak11720.jpg
+   :align: center
+   :alt: RAK11720
+
+Hardware
+********
+
+The easiset way to use a RAK11720, is the WisBlock Modular system.
+A WisBlock Base board (RAK19007) which provides the power
+supply and programming/debug interface is the base to plug a
+RAK11722 (WisBlock Core module with the RAK11720) in.
+
+- Apollo3 Blue SoC with up to 96 MHz operating frequency
+- ARM® Cortex® M4F core
+- 16 kB 2-way Associative/Direct-Mapped Cache per core
+- Up to 1 MB of flash memory for code/data
+- Up to 384 KB of low leakage / low power RAM for code/data
+- Integrated Bluetooth 5 Low-energy controller
+- Semtech SX1262 low power high range LoRa transceiver
+- iPEX connectors for the LORA antenna and BLE antenna.
+- 2 user LEDs on RAK19007 WisBlock Base board
+- Powered by either Micro USB, 3.7V rechargeable battery or a 5V Solar Panel Port
+
+For more information about the RAK11720 stamp module:
+
+- `WisDuo RAK11720 Website`_
+- `WisBlock RAK11722 Website`_
+
+Supported Features
+==================
+
+The RAK11720 module configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| MPU       | on-chip    | memory protection unit              |
```

Comment:
Generally driver/component column represents the path i.e `arch/arm` for MPU and likewise for all the rest.

Reply:
I got this table directly from 'boards/ambiq/apollo3_evb' index.rst file. I only added lora line to table. To be honest, I do not understand what I must change.

Reply:
Added with commit 279689e